proto/opamp.proto

```diff
@@ -635,7 +648,10 @@ enum AgentCapabilities {
     AgentCapabilities_ReportsHealth                  = 0x00000800;
     // The Agent will report RemoteConfig status via AgentToServer.remote_config_status field.
     AgentCapabilities_ReportsRemoteConfig            = 0x00001000;
-
+    // The Agent will report heartbeats on a default interval of 30s.
+    // This is specified by the ServerToAgent.OpAMPConnectionSettings.heartbeat_interval_seconds field.
+    // Status: [Beta]
+    AgentCapabilities_ReportsHeartbeat               = 0x00002000;
```

Comment:
Can you please clarify what the purpose of this capability flag is? How is the server logic going to be different depending on this flag?

Reply:
the server may set the `heartbeat_interval_seconds` field for the connection settings based on what the server knows its timeout may be.

Reply:
okay, can do.